src/main/GenomeTrack.js

```diff
@@ -27,7 +28,7 @@ var GenomeTrack = React.createClass({
     onRangeChange: React.PropTypes.func.isRequired,
   },
   render: function(): any {
-    return <div><canvas ref='canvas' /></div>;
+    return <canvas />;
   },
   componentDidMount: function() {
     var div = this.getDOMNode();
```

Comment:
You can in old-style React components (i.e. `React.createClass`) but not in new-style ES6 classes. With those you have to do `React.findDOMNode(this)`.